src/pytorch_lightning/trainer/call_fn.py

```diff
@@ -14,13 +14,14 @@
 import traceback
```

Comment:
Open to suggestions for a different name, but `teardown` is not the main responsibility of this function

Reply:
I would not rename the file if Will specifically called it this way.